service/src/functionalTest/java/uk/gov/hmcts/reform/fpl/StatelessCallbackApiTest.java

```diff
@@ -1,58 +1,28 @@
 package uk.gov.hmcts.reform.fpl;
 
-import com.fasterxml.jackson.databind.ObjectMapper;
-import io.restassured.RestAssured;
-import io.restassured.http.Header;
 import io.restassured.http.Headers;
-import net.serenitybdd.junit.spring.integration.SpringIntegrationSerenityRunner;
 import net.serenitybdd.rest.SerenityRest;
-import org.junit.Before;
 import org.junit.Test;
-import org.junit.runner.RunWith;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.boot.test.context.SpringBootTest;
-import org.springframework.test.context.ActiveProfiles;
-import org.springframework.test.context.ContextConfiguration;
 import uk.gov.hmcts.reform.fpl.model.Scenario;
 import uk.gov.hmcts.reform.fpl.model.User;
-import uk.gov.hmcts.reform.fpl.util.AuthenticationService;
-import uk.gov.hmcts.reform.fpl.util.ScenarioService;
-import uk.gov.hmcts.reform.fpl.util.TestConfiguration;
+import uk.gov.hmcts.reform.fpl.service.ScenarioService;
 
 import static com.gargoylesoftware.htmlunit.util.MimeType.APPLICATION_JSON;
-import static io.restassured.http.Headers.headers;
 import static java.lang.String.format;
+import static org.apache.commons.lang3.StringUtils.isEmpty;
+import static org.apache.commons.lang3.StringUtils.isNotEmpty;
 import static org.skyscreamer.jsonassert.JSONAssert.assertEquals;
 import static uk.gov.hmcts.reform.fpl.util.CallbackComparator.callbackComparator;
 import static uk.gov.hmcts.reform.fpl.util.StringUtils.blue;
 import static uk.gov.hmcts.reform.fpl.util.StringUtils.red;
 
 
-@SpringBootTest
-@RunWith(SpringIntegrationSerenityRunner.class)
-@ActiveProfiles("test-${env:local}")
-@ContextConfiguration(classes = {
-    TestConfiguration.class,
-    ObjectMapper.class,
-    ScenarioService.class,
-    AuthenticationService.class})
-public class CallbackTest {
-
-    @Autowired
-    private TestConfiguration testConfiguration;
+public class StatelessCallbackApiTest extends AbstractApiTest {
 
     @Autowired
     private ScenarioService scenarioService;
 
-    @Autowired
-    private AuthenticationService authenticationService;
-
-    @Before
-    public void setUp() {
-        RestAssured.baseURI = testConfiguration.getFplUrl();
-        RestAssured.useRelaxedHTTPSValidation();
-    }
-
```

Comment:
as discussed, that can be flattened and inline in some stateless abstract class, with the only parameter the input/output to simplify (or something like that)